src/db/models/store.js

```diff
@@ -0,0 +1,17 @@
+"use strict";
+module.exports = (sequelize, DataTypes) => {
+  const Store = sequelize.define(
+    "Store",
+    {
+      id: DataTypes.FLOAT,
+      name: DataTypes.STRING,
+      createdAT: DataTypes.DATE,
+      updatedAt: DataTypes.DATE,
+    },
+    {}
```

Comment:
if you are going to use timestamps, you need to declare timestamps: true in this object. Also, you change changed how createdAt and updatedAt are written if you decide to go with all underscored row names.
![Screen Shot 2021-01-18 at 6 17 50 PM](https://user-images.githubusercontent.com/31671880/104973144-7adc1e80-59b9-11eb-9c20-5e8236c05d52.png)